Dockerfile

```diff
@@ -22,6 +19,13 @@ RUN \
  apt autoremove --yes && \
  apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
+FROM ubuntu:bionic as oasis-chain-runner
+
+COPY --from=oasis-chain-builder /usr/local/bin/oasis-chain /usr/local/bin
```

Comment:
Probably copy from `/usr/local/lib/cargo/bin/oasis-chain`? You are copying symlink then?

Reply:
it's not a symlink. it's `cp`'d

Reply:
You are right. I do not know why I thought that `/usr/local/bin/oasis-chain` is a symlink.